src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -78,24 +69,33 @@ public void SaveConfig()
 
         public void SaveComponents()
         {
-            try
+            if (ComponentFileToDocumentMap.Any())
```

Comment:
Why the Any()? The foreach will skip anyways...?

Reply:
Will update.